src/pages/AddProperty.tsx

```diff
@@ -75,21 +102,50 @@ const AddProperty = () => {
         }
     }, [preview])
 
-    //called when submit button is pressed
-    const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
-        event.preventDefault();
-        //store image to db on submit
+    const replacePhoto = async () => {
         let url: string | void = "";
```

Comment:
Again this is an indication that something else is wrong. It's not good to have a variable that could either be a string or nothing, you have no idea!

You should be able to fix this by adding `return ""` (an empty string) in the `catch` function, then handling it appropriately later in the method (throw an error?).